app/services/batched_remove_status_service.rb

```diff
@@ -83,9 +75,9 @@ def unpush_from_list_timelines(account, statuses)
   end
 
   def unpush_from_public_timelines(status)
-    return unless status.public_visibility?
+    return unless status.public_visibility? || status.id < @status_id_cutoff
```

Comment:
Return if `!(status.id < @status_id_cutoff)`... So that's return if `status.id >= @status_id_cutoff`, I think it's the opposite of what you want, since it's statuses *above* the cutoff we want to stream. You can change the `unless` to `if` when conditions become more complicated.

Reply:
oh, indeed >_<